packages/flutter/test/widgets/binding_test.dart

```diff
@@ -76,6 +85,12 @@ class RentrantObserver implements WidgetsBindingObserver {
     WidgetsBinding.instance.addObserver(this);
   }
 
+  @override
+  void didChangeViewFocus(ViewFocusEvent event) {
```

Comment:
Should this get called from the "Rentrant observer callbacks do not result in exceptions" test?

Reply:
Done.